Update README with deployment instructions

cpeel opened the pull request:
Add how to install `ebookmaker-web`. The `config.php` example is an example and not the exact file used on infero.

Really `$tmpdir` and `$mybaseurl` should point to the same location but I'm not solving this in this documentation PR.

cpeel committed:
Update README with deployment instructions

gbnewby committed:
Update README.md

Added an example cron line.

gbnewby commented:
This all looks fine, and there are many improvements. 

Per some other comments, let's give Eric a chance to spin this up, prior to deploying to production. 

There's no reason to wait with committing all proposed changes to the dev branch, though.

eshellman commented:
THANK YOU

eshellman commented:
what version of PHP?

cpeel commented:
> what version of PHP?

Inferno is running 8.3 (Ubuntu 24.04) but this should probably work with any version >= 7.4.

cpeel commented:
> > what version of PHP?
> 
> Inferno is running 8.3 (Ubuntu 24.04) but this should probably work with any version >= 7.4.

Correction: Inferno is running 8.1 (Ubuntu 22.04).

cpeel committed:
Update README

Include web server configuration information on viewing output
directory contents and improve cronjob command.
